lms/djangoapps/lti_provider/models.py

```diff
@@ -0,0 +1,14 @@
+"""
+Database models for the LTI provider feature.
+"""
+from django.db import models
+
+
+class LtiConsumer(models.Model):
+    """
+    Database model representing an LTI consumer. This model stores the consumer
+    specific settings, such as the OAuth key/secret pair and any LTI fields
+    that must be persisted.
+    """
+    key = models.CharField(max_length=32, unique=True, db_index=True)
+    secret = models.CharField(max_length=32, unique=True)
```

Comment:
Can you please add an admin.py for this so these values are easier to access? It can be a trivial one, along the lines of:

```python
from django.contrib import admin

from .models import LTIConsumer

admin.site.register(LTIConsumer)
```